packages/docs/components/SideNav/SideNav.tsx

```diff
@@ -30,7 +30,7 @@ export const SideNav: React.FC = () => {
         <SideNavGroup title="Introduction">
           <SideNavLink href="/">Getting Started</SideNavLink>
 
-          <Link title="CodeSandbox Example" url={CodeSandboxUrl} />
+          <Link title="CodeSandbox Example" url={String(CodeSandboxUrl)} />
```

Comment:
This will be a `"undefined"` if it's undefined. Should we just null coalesce to an empty string?